.github/workflows/ci.yml

```
@@ -232,20 +193,96 @@ jobs:
       env:
         NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
     
-    - name: Commit version bump and create Git tag
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v3
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)